File: VBNET_AutoMapper/Aluno.vb
Public Class Aluno
    Public Property Nome() As String
    Public Property Email() As String
    Public Property Endereco() As Endereco
    Public Property Genero() As String
    Public Property Idade() As Integer
    Public Property Nascimento() As DateTime
End Class

Public Class Endereco
    Public Property Pais() As String
    Public Property Cidade() As String
    Public Property Rua() As String
    Public Property Cep() As String
End Class

Public Class AlunoViewItem
    Public Property Nome() As String
    Public Property Email() As String
    Public Property Endereco() As String
    Public Property Genero() As String
    Public Property Idade() As Integer
    Public Property Nascimento() As String

End Class

File: CSharp_AutoMapper/Aluno.cs
using System;

namespace CSharp_AutoMapper
{
    public class Aluno
    {
        public string Nome { get; set; }
        public string Email { get; set; }
        public Endereco Endereco { get; set; }
        public string Genero { get; set; }
        public int Idade { get; set; }
        public DateTime Nascimento { get; set; }
    }

    public class Endereco
    {
        public string Pais { get; set; }
        public string Cidade { get; set; }
        public string Rua { get; set; }
        public string Cep { get; set; }
    }

    public class AlunoViewItem
    {
        public string Nome { get; set; }
        public string Email { get; set; }
        public string Endereco { get; set; }
        public string Genero { get; set; }
        public int Idade { get; set; }
        public string Nascimento { get; set; }
    }

  
}
